CriteoPublisherSdk/Sources/Network/Serializers/CR_BidRequestSerializer.m

```diff
@@ -123,4 +123,62 @@ - (NSArray *)slotsWithCdbRequest:(CR_CdbRequest *)cdbRequest {
   return slots;
 }
 
++ (NSDictionary<NSString *, id> *)mergeToNestedStructure:
```

Comment:
This code depending on client data, which itself could rely on some database or else.
That could easily crash once in production, breaking any bid attempt as I see it will later be used by the `ApiHandler`.
I would strongly suggest to `@try` / `@catch` there with logs / degraded mode fallback value.

Reply:
Values are just copied, so even if there is garbage, I don't see why it could fail. But ok, I've done it